.buildkite/template.yml

```diff
@@ -1,7 +1,9 @@
-# template.yml
-# Use this as a starting point for your own Buildkite pipeline
 name: "Artifacts Pipeline Example"
-description: "An example repository you can use as a test project with Buildkite"
+description: "An example pipeline that generates and downloads artifacts across build steps."
+emoji: ":package:"
+color: "gray-200"
```

Comment:
Could we default to `gray-100`? That's what I have in the designs.

BTW generally the `100` colours work well for these (with the exception of green, where `200` works better)